client/web/src/enterprise/batches/list/BatchChangeListPage.story.tsx

```diff
@@ -38,6 +38,14 @@ const config: Meta = {
 
 export default config
 
+window.context.licenseInfo = {
+    currentPlan: 'team-0',
+    batchChanges: {
+        maxNumChangesets: 10,
+        unrestricted: false,
```

Comment:
We might want to make this an `unrestricted` plan, too, since I think that would have been the default behavior before the context changes:

```suggestion
    currentPlan: 'enterprise-0',
    batchChanges: {
        maxNumChangesets: -1,
        unrestricted: true,
```

Reply:
(Bonus points if you want to add a *new* story for a restricted license plan, though! 😛)